Sources/DictionarySpecs/dictionaryManagerSpec.swift

```diff
@@ -0,0 +1,14 @@
+import Dictionary
+import Swiftest
+
+class DictionaryManagerSpec: Swiftest.Spec {
+	let spec = describe("dictionary logic") {
+		describe("#getRandomWord") {
+			it("returns a random word from the dictionary") {
+				let dictionaryManager = DictionaryManager()
+				let word = dictionaryManager.getRandomWord()
+				expect(dictionaryManager.gameWords).to.contain(word)
+			}
```

Comment:
There is a test for the happy path...but, I was wondering if there is an unhappy path scenario that should be tested...There may not be one and I'm being confusing :smile: